### PR TITLE
[apptools-android] Update tests for apptools run on windows

### PR DESCRIPTION
**Pull request by yunxliu**
Impacted tests(approved): new 0, update 66, delete 0
Unit test platform: [Android]
Unit test result summary: pass 62, fail 4, block 0

https://crosswalk-project.org/jira/browse/XWALK-4032

### CODE REVIEW

File: apptools/apptools-android-tests/README [hunk 1]
```diff
@@ -3,21 +3,38 @@
...
+3. Need to edit the file "apptools-android-tests/host.txt" according to the type of test host.
+If test host is "Windows" platform, content of the file should be "Windows", default value is "Android"
+4. Need to edit the file "apptools-android-tests/version.txt" according to the Crosswalk which you want to test.
 For example: if you want to test with "crosswalk-13.42.319.11.zip", content of the file should be "13.42.319.11"
-4. The node.js, Android SDK, and git must be functional
-5. Install nodeunit: sudo npm install nodeunit -g
+5. The node.js, Android SDK, JDK, apache ant, and git must be functional
 6. Download crosswalk-app-tools to apptools-android-tests/tools, then install npm
   6.1 cd tools, then clone source code from https://github.com/crosswalk-project/crosswalk-app-tools
   6.2 Run commands: cd crosswalk-app-tools, then run: sudo npm install
```
Comment (Honry) [inline apptools/apptools-android-tests/README]: No 'sudo' cmd in Windows platform

File: apptools/apptools-android-tests/README [hunk 2]
```diff
@@ -3,21 +3,38 @@
...
-7. Environment variable configuration:
-  7.1 export ANDROID_HOME=$(dirname $(dirname $(which android)))
-  7.2 export CROSSWALK_APP_TOOLS_CACHE_DIR=/path/to/opt/apptools-android-tests/tools
+7. Download BeautifulSoup from http://www.crummy.com/software/BeautifulSoup/bs4/download/4.4/beautifulsoup4-4.4.0.tar.gz and install it
```
Comment (Honry) [inline apptools/apptools-android-tests/README]: Can it be installed by cmd?

File: apptools/apptools-android-tests/apptools/comm.py [hunk 3]
```diff
@@ -57,6 +61,15 @@ def setUp():
         ARCH = "x86"
     fp.close()
 
+    host = open(ConstPath + "/../host.txt", 'r')
+    if host.read().strip("\n\t") != "android":
+        HOST = "node "
```
Comment (Honry) [inline apptools/apptools-android-tests/apptools/comm.py]: Better name as HOST_PREFIX

File: apptools/apptools-android-tests/README (hunk 2 above)
Comment (yunxliu) [inline apptools/apptools-android-tests/README]: Yes, it can be installed by cmd

**Comment by Honry** [comment]
Thanks, @yunxliu. LGTM

**Pull request merged**